Assignment6.Rmd

```diff
@@ -70,17 +75,29 @@ table(M2$certified, M2$predict1)
 
 table(M2$certified, M2$predict2)
 
+
+#The second tree has a lower error rate
```

Comment:
What does this suggest?

Reply:
as far as I know, the tree with the lower rate gives us a better prediction.